prov/efa/src/rxr/rxr_pkt_entry.h

```diff
@@ -41,17 +41,18 @@ enum rxr_pkt_entry_state {
 	RXR_PKT_ENTRY_FREE = 0,
 	RXR_PKT_ENTRY_IN_USE,
 	RXR_PKT_ENTRY_RNR_RETRANSMIT,
-	RXR_PKT_ENTRY_COPY_BY_READ, /* entries contain data. A read request has been posted to copy
-				     * to receiving buffer
-				     */
+	RXR_PKT_ENTRY_READING, /* entries contain data. A read request has been posted to copy
```

Comment:
Any particular reason for the name change? The old name is clearer to me.

Reply:
I had to introduce a new packet type and I named it ```RXR_PKT_ENTRY_READ_COPY```, I think the old name ```RXR_PKT_ENTRY_COPY_BY_READ``` will be confused with the new type, so I renamed it ```RXR_PKT_ENTRY_READING``` because reading is more like a state.